libraries/AP_BattMonitor/AP_BattMonitor_Generator.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "AP_BattMonitor.h"
+#include "AP_BattMonitor_Backend.h"
+
+#include <AP_Generator/AP_RichenPower.h>
```

Comment:
I wonder if perhaps we should name the class AP_Generator_RichenPower assuming that we add support for more generators in the future.